tests/std/include/range_algorithm_support.hpp

```diff
@@ -122,6 +122,26 @@ namespace test {
             ptr_ = s.peek();
         }
         // clang-format on
+
+        friend constexpr boolish operator==(sentinel const s, Element* const ptr) noexcept {

```

Comment:
Should these operators be `[[nodiscard]]`?

Reply:
Yes. There a ton of operators in here that should be `[[nodiscard]]`.